cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/MockKube.java

```diff
@@ -379,6 +384,33 @@ protected void nameScopedMocks(RollableScalableResource<StatefulSet, DoneableSta
                         //        pod);
                         mockPods.inNamespace(argument.getMetadata().getNamespace()).withName(podName).create(pod);
                         addPodRestarter(mockPods, resourceName, podNum, podName);
+
+                        if (value.getSpec().getVolumeClaimTemplates().size() > 0) {
+
+                            for (PersistentVolumeClaim pvcTemplate: value.getSpec().getVolumeClaimTemplates()) {
+
+                                String pvcName = pvcTemplate.getMetadata().getName() + "-" + podName;
+                                if (mockPvcs.inNamespace(argument.getMetadata().getNamespace()).withName(pvcName).get() == null) {
+
+                                    Map<String, String> labels = new HashMap<>();
+                                    labels.put(Labels.STRIMZI_KIND_LABEL, Kafka.RESOURCE_KIND);
+                                    labels.put(Labels.STRIMZI_CLUSTER_LABEL, value.getMetadata().getLabels().get(Labels.STRIMZI_CLUSTER_LABEL));
+                                    labels.put(Labels.STRIMZI_NAME_LABEL, argument.getMetadata().getName());
+
+                                    LOGGER.debug("create Pvc {} because it's in StatefulSet {}", pvcName, resourceName);
```

Comment:
```suggestion
                                    LOGGER.debug("create Pvc {} because it's in VolumeClaimTemplate of StatefulSet {}", pvcName, resourceName);
```